packages/stripe-graphql/src/schema/customer.ts

```diff
@@ -0,0 +1,66 @@
+import { builder } from '../builder'
+
+import { PaymentMethodTypes } from './payment-methods'
+
+builder.objectType('StripeCustomer', {
+  description:
+    'This object represents a customer of your business. It lets you create recurring charges and track payments that belong to the same customer.',
+  fields: (t) => ({
+    id: t.exposeString('id'),
+    object: t.exposeString('object'),
+    balance: t.exposeInt('balance'),
+    name: t.exposeString('name', {
+      nullable: true
+    }),
+    address: t.expose('address', {
+      type: 'StripeAddress',
+      nullable: true
+    }),
+    paymentMethods: t.field({
+      type: 'StripePaymentMethods',
+      args: {
+        type: t.arg({
+          type: PaymentMethodTypes,
+          required: true,
+          defaultValue: 'card'
+        }),
+        startingAfter: t.arg.string({
+          required: false
+        }),
+        endingBefore: t.arg.string({
+          required: false
+        }),
+        limit: t.arg.int({
+          required: false
+        })
+      },
+      nullable: false,
+      resolve: async (customer, { type, startingAfter, endingBefore, limit }, { stripe }) => {
```

Comment:
This could be implemented as a relay connection instead (using the relay plugin):

```ts
t.connection({
  type: 'StripePaymentMethod',
  args: {
    type: t.arg({
      type: PaymentMethodTypes,
      required: true,
      defaultValue: 'card',
    }),
  },
  resolve: (customer, { first, last, before, after, type }) => {
    const paymentMethods = await stripe.customers.listPaymentMethods(customer.id, {
      type,
      starting_after: after || undefined,
      ending_before: before || undefined,
      limit: before ? last : first,
    });

    return {
      pageInfo: {
        hasNextPage: paymentMethods.has_more,
        hasPreviousPage: !!after,
        startCursor: paymentMethods.data[0]?.id,
        endCursor: paymentMethods.data[paymentMethods.data.length - 1]?.id,
      },
      edges: paymentMethods.data.map((paymentMethod) => ({
        cursor: paymentMethod.id,
        node: paymentMethod,
      })),
    };
  },
});
```

Reply:
Would this mean we would "break" the Stripe's API convention? 

I'm trying to keep the API similar to what Stripe's API returns, except for transforming snake_case to camelCase.

Reply:
yes this would be aligning with graphql/relay conventions over stripe conventions.  I don't know if that is a good tradeoff here, and sticking with stripes conventions may be more valuable.  The relay connection pattern can be useful because many clients have support for automatically paginating lists of this shape, which can make it easier to build reusable components/libraries that will work with various data sources.  I don't have strong opinions here, just figured I'd mention it just in case. 

https://relay.dev/graphql/connections.htm

Reply:
Thanks. For now, I think we'll stick to Stripe's convention, so we don't add too much abstraction on top of Stripe's API. Might come back to this at a later stage.